### PR TITLE
Add YAO to Easypanel 🎉 

### PR DESCRIPTION
**Pull request by Supernova3339**

### CODE REVIEW

File: templates/yao/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - appServiceImage
+    - databaseServiceName
+  properties:
+    projectName:
+      type: string
+      title: Project Name
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: wordpress
```
Comment (Ponkhy) [inline templates/yao/meta.yaml:31]:
```suggestion
      default: yao
```

**Review by Ponkhy — APPROVED** [review]
LGTM

File: templates/yao/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+name: YAO
+description: YAO is an open-source application engine,
+instructions: Use xiang@iqka.com:A123456p+ to login at /xiang/login/admin . Use amd64 instead of arm64 if you use a non ARM based system.
```
Comment (deiucanta) [inline templates/yao/meta.yaml:3]: is there a way to change the default credentials using env vars?

Comment (Supernova3339) [reply templates/yao/meta.yaml:3]: this is hardcoded sadly and there is no way to change it as of now.

**Pull request merged**